pandevice/base.py

```diff
@@ -2775,10 +2781,10 @@ class XapiWrapper(pan.xapi.PanXapi):
         def __init__(self, *args, **kwargs):
             self.pan_device = kwargs.pop('pan_device', None)
             pan.xapi.PanXapi.__init__(self, *args, **kwargs)
-
+            pred = lambda x: inspect.ismethod(x) or inspect.isfunction(x)
```

Comment:
Got it, thanks!  Can you add a comment to the code indicating why it is handled this way?  Just a note about what is needed for python 2 and for python 3.  Thanks!